site/public/templates/grading/OverallComment.twig

```diff
@@ -31,13 +31,13 @@ Required Parameters:
             markdown_area_id : "overall-comment-" ~ (overall_comment["logged_in_user"]["user_id"]|escape) ,
             markdown_area_name : '',
             markdown_area_value : (overall_comment["logged_in_user"]["comment"]|escape),
-            class : "general-comment-entry noscroll key_to_click",
+            root_class : "general-comment-entry",
             placeholder : "Overall message for student about the gradeable...",
             preview_div_id : "overall-comment-markdown-preview-" ~ (overall_comment["logged_in_user"]["user_id"]|escape),
             preview_div_name : "overall-comment-markdown-preview-" ~ (overall_comment["logged_in_user"]["user_id"]|escape),
             preview_button_id : "overall-comment-markdown-preview-button",
-            onclick : "previewOverallCommentMarkdown.call(this, '" ~ (overall_comment["logged_in_user"]["user_id"]|escape) ~ "')", 
-            render_buttons : true,
+            onclick : "previewOverallCommentMarkdown.bind(this, '" ~ (overall_comment["logged_in_user"]["user_id"]|escape) ~ "')", 
+            render_header : true,
             min_height : "100px",
             max_height : "150px",
```

Comment:
Should the min/max height parameters (elsewhere too, not just here) be measured in `em` units instead of `px` units?  If you measure it in `em` units you can control roughly how many lines tall the box is.  The counterargument here is that everything else is measured in pixels so why not do it here.  I'm not sure what the best option is here but I wanted to throw it out there just in case it was useful.

Reply:
You're right about `em` being a better solution. It was simply a case of me converting the various forms throughout the site to use this common template, and it just so happens that those pages used `px` to define sizes so I did too since it was just easier. I'll add that conversion to my list of things to do later